components/tools/OmeroWeb/omeroweb/webgateway/templates/webgateway/viewport/omero_image.html

```diff
@@ -384,7 +403,18 @@
       if (viewport.viewportimg.get(0).hide_rois) {
           viewport.viewportimg.get(0).hide_rois();
       }
+
+      // Disable the "Hide ROIs" button ...
+      $("#hide-rois-a").off("click");
+      $("#hide-rois-a").removeAttr("href");
+      $("#hide-rois-a").css("color", "gray");
+      // ... and enable the "Show ROIs" button
+      $("#show-rois-a").on("click", function(){show_rois(); return false;});
+      $("#show-rois-a").attr("href", "#");
+      $("#show-rois-a").css("color", "");
```

Comment:
Just a minor style comment (suggestion, not blocker)...
To avoid creating the jQuery element on every line, it's sometime nicer to chain calls like
```
$("#hide-rois-a").off("click")
    .removeAttr("href")
    .css("color", "gray")
    .on("click", function(){show_rois(); return false;})
    .attr("href", "#")
    .css("color", "");
```
or if you're going to use the element repeatedly in various places, just cache it.
```
var $hide_rois_a = $("#hide-rois-a");
$hide_rois_a.off("click");
$hide_rois_a.removeAttr("href");
$hide_rois_a.css("color", "gray");
$hide_rois_a.on("click", function(){show_rois(); return false;});
$hide_rois_a.attr("href", "#");
$hide_rois_a.css("color", "");
```